packages/generator/src/artifacts/js/is-valid-prop.ts

```diff
@@ -10,5 +11,11 @@ export function generateisValidProp(ctx: Context) {
   )
   return {
     js: content,
+    ts: outdent`
```

Comment:
To be consistent with others

```suggestion
    dts: outdent`
```